addons/payment_xendit/static/src/js/payment_form.js

```diff
@@ -121,7 +121,7 @@ paymentForm.include({
                 ...this._xenditGetPaymentDetails(paymentOptionId),
                 // Allow reusing tokens when the users wants to tokenize.
                 is_multiple_use: this.paymentContext.tokenizationRequested,
-                amount: processingValues.amount,
+                amount: processingValues.rounded_amount,
```

Comment:
Equivalent but it helps suppress warnings with some JS linters:
```suggestion
                amount: processingValues['rounded_amount'],
```